src/Components/MovieForm/MovieForm.jsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+import styles from './MovieForm.module.css';
+import { LabeledInput } from './LabeledInput/LabeledInput';
+
+export const MovieForm = ({ initialValues = {}, onSubmit, genres }) => {
+  const [movie, setMovie] = useState({
+    title: initialValues?.title ?? '',
+    releaseDate: initialValues?.releaseDate ?? '',
+    imageUrl: initialValues?.imageUrl ?? '',
+    rating: initialValues?.rating ?? '',
+    genre: initialValues?.genre ?? '',
+    runtime: initialValues?.runtime ?? '',
+    description: initialValues?.description ?? '',
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setMovie({ ...movie, [name]: value });
+  };
+
+  const handleReset = () => {
+    setMovie({
+      title: '',
+      releaseDate: '',
+      imageUrl: '',
+      rating: '',
+      genre: '',
+      runtime: '',
+      description: '',
+    });
+  };
+
+  return (
+    <form
+      className={styles.container}
+      onSubmit={(e) => {
+        e.preventDefault();
+        const formData = Object.fromEntries(new FormData(e.target));
+        onSubmit(formData);
+      }}
+    >
+      <div className={styles.formRow}>
+        <div className={styles.formGroup}>
+          <LabeledInput label={'Title'} onChange={handleChange} value={movie.title} id={'title'} />
```

Comment:
```suggestion
          <LabeledInput label='Title' onChange={handleChange} value={movie.title} id='title' />
```

And in other places like this.